script/analysis/snap.py

```diff
@@ -101,6 +119,10 @@ def make_snap(dfnam,vnam,coords,size,cmap,logplot,
       bplt.plot_xz(ax, geom, var, dump, cmap=cmap, vmin=vmin, vmax=vmax,
         cbar=False, label=label, ticks=None, shading='gouraud')
       ax.set_xlim([-size,size]); ax.set_ylim([-size,size])
+    if args.A_contours:
```

Comment:
I just noticed that args is referenced in `make_snap` now. This is going to break `movie.py` which calls this method. Can you pass these through as optional arguments to `make_snap` so that the args object isn't used in scope it might not be available?